docs/to_clarify.md

```diff
@@ -0,0 +1,8 @@
+## Question
+1. What are the existing solutions that was mentioned in point 2 of the "Possible Use Cases"
+
+1. What are the digital solutions that the industry have tried? To potentially prevent designing similar solutions 
+
+1. Is there any other activities that depend on the current check-in/out system. In the case when solution replaces current check-in/out systems do we need to cater for these dependencies? For example, if a check-in kiosk is currently used, and it has maps to guide guests after checking in, this would mean our solution would also need to include guiding guests after checking in 
+
+1. Can we run through the current check-in/out process? 
```

Comment:
This question can be better reworded and we can add the purpose of the question: Can **you** run through **with us** the current check-in/out process? This will help us have a better understanding of the pain points experienced when using the existing solution.